smartsim/_core/mli/infrastructure/storage/dragonfeaturestore.py

```diff
@@ -26,26 +26,29 @@
 
 import typing as t
 
-import smartsim.error as sse
-from smartsim._core.mli.infrastructure.storage.featurestore import FeatureStore
-from smartsim.log import get_logger
+# pylint: disable=import-error
+# isort: off
+import dragon.data.ddict.ddict as dragon_ddict
 
-if t.TYPE_CHECKING:
-    from dragon.data.ddict.ddict import DDict
+# isort: on
 
+from smartsim._core.mli.infrastructure.storage.featurestore import FeatureStore
+from smartsim.error import SmartSimError
+from smartsim.log import get_logger
 
 logger = get_logger(__name__)
 
 
 class DragonFeatureStore(FeatureStore):
     """A feature store backed by a dragon distributed dictionary"""
 
-    def __init__(self, storage: "DDict") -> None:
+    def __init__(self, storage: "dragon_ddict.DDict") -> None:
```

Comment:
Is ``:param:`` needed for ``storage``

Reply:
Added `:param:` and `:raises:` where appropriate